### DISCUSSION TITLE
Is there any special handling for CTRL+F in synedit?

### DISCUSSION THREAD

**Discussion by ClementDoss** [TurboPack/SynEdit#199] [General]
Hi,

I hooked SynEdit OnKeyDown for a special action when CTRL+F is pressed, but the event is never triggered.
Other key combinations work fine, for instance: CTRL+SHIFT+F, ALT+F, CTRL+G, CTRL+H are triggering onKeyDown event except CTRL+F. I want to display a Search Dialog when this combination is pressed.

Thanks

**Comment by pyscripter** [comment]
https://www.google.com/search?q=ctrl+f+not+working

**Reply by ClementDoss** [reply]
It's not working with SynEdit. Every ohter application ( Delphi, Firefox, Google, Notepad++, etc) it's working fine.
I had a preview version of power toys, uninstalled it. But still no joy.
Synedit is docked on a frame. Every other key combination works fine, except for CTRL+F

**Reply by MShark67** [reply]
I use Ctrl-F with SynEdit, but note that I use an action shortcut and not an OnKeyPress handler.  As far as I can tell, SynEdit doesn't use Ctrl-F for any of its built-in keystrokes.  Have you checked to see if Ctrl-F is being used in any of your shortcut handlers such that onKeyPress isn't getting those keys?

**Reply by ClementDoss** [reply]
I've been looking for a handler in my code, but have found nothing so far. As I'm using some of SynEdit dialog, I thought maybe CTRL+F was used over there.. Anyway, I'll keep looking in my code

Thanks

**Reply by ClementDoss** [reply]
Found it!
If I assign a TPopupMenu to TSynedit.Popupmenu some shortcut keys just stop working. I have removed the popupmenu assignment and everything is working as expected.
Still haven't found why.... I do still have a lot of coffee

Anyway, I don't think it is related to TSynedit. I will try to duplicate in a small sample.
